docs/bbs.md

```diff
@@ -869,6 +869,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/newsmth/account/fef705ec94819a5a87941759e33c0982" path="/newsmth/account/:id" :paramsDesc="['用户 id，可在用户页的 URL 中找到']"/>
 
+## 司机社
+
+### 论坛
+
+<Route author="akynazh" example="/sjs/forum/51" path="/sjs/forum/:fid" :paramDesc="['子论坛 id']">
```

Comment:
Please update path to `/xsijishe/forum/:fid`

Reply:
done